src/scripts/main.js

```diff
@@ -1,3 +1,222 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+
+let table, headers, tbody, rows;
+
+const init = () => {
+  table = document.querySelector('table');
+  headers = [...table.querySelectorAll('th')];
+  tbody = table.querySelector('tbody');
+  rows = [...tbody.querySelectorAll('tr')];
+};
```

Comment:
It is a good practice to avoid using global variables as much as possible. The `init` function could be improved by returning an object with the `table`, `headers`, `tbody`, and `rows` variables. This would allow you to avoid using global variables and make your code more predictable and easier to debug.